### PR TITLE
Filter datasets by its datasource

### PR DESCRIPTION
**Pull request by davidebukali**
Add a drop down with datasources on `MyDatasets` and `PublishedDatasets` that filters datasets by selected datasource #352

### CODE REVIEW

File: frontend/src/components/OperationsTableCard/OperationsTableCard.tsx [hunk 1]
```diff
@@ -50,12 +60,26 @@ const OperationsTableCard: FunctionComponent<OperationsTableCardProps> = (props)
   const [showMyQueries, setShowMyQueries] = useState(props.showMyQueries);
   const [searchQuery, setSearchQuery] = useState('');
   const [info, setInfo] = useState('');
+  const [dropDownValues, setDropDownValues] = useState<DropdownItemProps[]>([]);
   const onModalHide = () => setInfo('');
+  // const { sources } = useContext(SourcesContext);
+  const { sources } = useContext(SourcesContext);
```
Comment (edwinmp) [inline frontend/src/components/OperationsTableCard/OperationsTableCard.tsx]: ?

File: frontend/src/components/OperationsTableCard/OperationsTableCard.tsx [hunk 2]
```diff
@@ -50,12 +60,26 @@ const OperationsTableCard: FunctionComponent<OperationsTableCardProps> = (props)
...
 
   useEffect(() => {
     fetchQueries(showMyQueries);
   }, []);
 
+  useEffect(() => {
+    console.log(`OperationTableCard ${sources}`);
```
Comment (edwinmp) [inline frontend/src/components/OperationsTableCard/OperationsTableCard.tsx]: ??

**Review by edwinmp — CHANGES_REQUESTED** [review]
image
Need these on one line and the dropdown filter must allow to be cleared... sure there's an option for that

File: frontend/src/components/OperationsTableCard/OperationsTableCard.tsx (hunk 2 above)
Comment (davidebukali) [reply frontend/src/components/OperationsTableCard/OperationsTableCard.tsx]: @edwinmp Removed console logs and updated dropdown to be next to text-search input

**Comment by davidebukali** [comment]
@edwinmp sorry had left out `clearable` datasources dropdowns, just added in the last commit.

**Review by edwinmp — APPROVED** [review]

**Pull request merged**